### PR TITLE
Us27 show hours on hover

### PR DESCRIPTION
**Pull request by mateuszs-kainos**
Add tooltip with hours on recruiter view table.

### CODE REVIEW

File: src/main/resources/assets/js/application/recruiter/views/tdpr-directive-slot.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <td class="slot" ng-class="getClass()">
-    <span class="slot-span" uib-tooltip="{{slotData.type}}"></span>
-</td>
\ No newline at end of file
+    <span class="slot-span" uib-tooltip="{{getTooltipString()}} {{slotData.type}}"></span>
```
Comment (jraniszewski) [inline src/main/resources/assets/js/application/recruiter/views/tdpr-directive-slot.html]: Could we put slotData.type to getTooltipString() function? Would be easier to manipulate it later.

Comment (mateuszs-kainos) [inline src/main/resources/assets/js/application/recruiter/views/tdpr-directive-slot.html]: Done.

File: src/main/resources/assets/js/application/recruiter/directives/tdprSlotDirective.js [hunk 2]
```diff
@@ -5,11 +5,16 @@ define(['angular', 'application/recruiter/tdprRecruiterModule'], function (angul
...
             replace: true,
             scope: {
-                slotData: '='
+                slotData: '=',
+                slotTimeData: '='
             },
             link: function (scope, element, attributes) {
                 scope.getClass = function () {
-                  return scope.slotData ? 'cell-avail-' + scope.slotData.type : '';
+                    return scope.slotData ? 'cell-avail-' + scope.slotData.type : '';
```
Comment (RogowskaMalgorzata) [inline src/main/resources/assets/js/application/recruiter/directives/tdprSlotDirective.js:13]: Why tab here?

Comment (marcint-kainos) [inline src/main/resources/assets/js/application/recruiter/directives/tdprSlotDirective.js:13]: Auto formatting probably

**Comment by rlewan** [comment]
Please update this branch with changes from master and I can merge 👍

**Pull request merged**